src/Raven.Server/Config/Categories/DatabaseConfiguration.cs

```diff
@@ -13,29 +13,29 @@ public class DatabaseConfiguration : ConfigurationCategory
         public TimeSetting OperationTimeout { get; set; }
 
         /// <summary>
-        /// This much time has to wait for the resource to become available when too much
-        /// different resources get loaded at the same time
+        /// This much time has to wait for the database to become available when too much
+        /// different datavases get loaded at the same time
         /// </summary>
         [Description("The time in seconds to wait for a database to start loading when under load")]
         [DefaultValue(10)]
         [TimeUnit(TimeUnit.Seconds)]
-        [ConfigurationEntry("Databases.ConcurrentResourceLoadTimeoutInSec", ConfigurationEntryScope.ServerWideOnly)]
-        public TimeSetting ConcurrentResourceLoadTimeout { get; set; }
+        [ConfigurationEntry("Databases.ConcurrentDatabaseLoadTimeoutInSec", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
We are already in `Databases.` so maybe ConcurrentLoadTimeoutInSec would be enough?